srttools/io.py

```diff
@@ -741,7 +767,10 @@ def read_data(fname):
 
 def root_name(fname):
     """Return the file name without extension."""
-    return os.path.splitext(fname)[0]
+    fn, ext = os.path.splitext(fname)
+    if 'fits' in ext and not ext.endswith('fits'):
+        fn += ext.replace("fits", "").replace(".", "")
+    return fn
```

Comment:
With this modification, the monitor output images of a `.fits5` file becomes `latest5_10.ext` and `latest5_11.ext` instead of just `latest_10.ext` and `latest_11.ext`. Was this intended or not? Because with the new file names the `index.html` web page will not display any image (it only looks for `latest_x[x].[ext]` format). If this was intended, we should also modify the monitor behavior to let it copy/move the images with the correct file name.

Reply:
The image processing always outputs images starting with index 0, disregarding which feed(s) are actually present in the fits file. So, with the premise that a fits0 file will always contain only the feed 0, a fits1 file will always contain only the feed 1, etc, and knowing that the 7 (when using 7 feeds) fitsX files are always generated and saved around the same instant, we can follow this rule:
.fits3 -> originally outputs latest_0 and latest_1, we strip the number and assign 3 (from the extension) * 2 + original image index. In this way we get latest_6 and latest_7. This behavior leads to the same set of images the monitor outputs with a merged .fits file containing all the 7 feeds. I will take care of the images renaming process monitor-side, and will leave the io.py like it currently is. Editing io.py to remove the feed index from the name actually leads to a wrong behavior of the SDTpreprocess. Imagine if you execute:
`SDTpreprocess scan.fits1 scan.fits2 ...`
The process will at first output scan_0.img and scan_1.img for the .fits1 file, then it will output again scan_0.img and scan_1.img for the .fits2 file, overwriting the previous images. We have to avoid this.